.github/workflows/checks.yml

```diff
@@ -151,25 +189,23 @@ jobs:
           sudo sysctl -w kernel.apparmor_restrict_unprivileged_userns=0
 
       - name: Setup Python & Poetry Environment
-        uses: exasol/python-toolbox/.github/actions/python-environment@0.20.0
+        uses: exasol/python-toolbox/.github/actions/python-environment@1.1.0
         with:
-          poetry-version: 2.0.1
           python-version: ${{ matrix.python-version }}
+          poetry-version: 2.1.2
 
-      - name: Calculate Test Coverage
-        run: poetry run nox -s test:coverage
-        env:
-          TEST_DB_VERSION: ${{matrix.exasol-version}}
-          PYTEST_ADDOPTS: >
-            -W 'ignore::DeprecationWarning:luigi:'
-            -W 'ignore::DeprecationWarning:pkg_resources:'
-            -W 'ignore:pkg_resources is deprecated as an API:DeprecationWarning'
-            -W 'ignore:Deprecated call to \`pkg_resources.declare_namespace:DeprecationWarning'
-            -W 'ignore::DeprecationWarning:exasol_integration_test_docker_environment:'
+      - name: Run Tests and Collect Coverage
+        run: |
+          poetry run -- nox -s test:unit -- --coverage \
```

Comment:
Sanity  check: want `test:unit` instead of ` test:coverage` here?

Reply:
If I remember, we changed the test:coverage to run only unit tests. That means we can change to test:coverage nowz before it ran integration tests, too, which is too expensive

Reply:
No, it looks like it does both https://github.com/exasol/python-toolbox/blob/main/exasol/toolbox/nox/_test.py#L99. Also, it doesn't seem like notebook-connector overrides it.